app/code/Magento/InventoryInStorePickupQuote/registration.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+\Magento\Framework\Component\ComponentRegistrar::register(
+    \Magento\Framework\Component\ComponentRegistrar::MODULE,
+    'Magento_InventoryInStorePickupQuote',
+    __DIR__
+);
```

Comment:
Could I ask you to add the new line at the end of file everywhere:))

Reply:
Already in place.